server/utils/storage/google_cloud_storage_client.go

```
@@ -132,6 +133,41 @@ func (g *GoogleCloudStorageClient) UploadPublicObject(ctx context.Context, bucke
+		w.ContentType = contentType
+	}
+	if cacheControl != "" {
+		w.CacheControl = cacheControl
+	}
+
+	if !attrs.UniformBucketLevelAccess.Enabled {
+		w.PredefinedACL = "publicRead"
+	}
+
+	if _, err := io.Copy(w, fileReader); err != nil {
+		w.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `w.Close` is not checked

(errcheck)